addons/website/models/website.py

```diff
@@ -1546,18 +1546,19 @@ def _get_cached_values(self):
     def _get_cached(self, field):
         return self._get_cached_values()[field]
 
-    def _get_html_fields_attributes_blacklist(self):
+    def _get_html_fields_blacklist(self):
         return (
             'mail.message', 'mail.activity', 'digest.tip',
         )
 
-    def _get_html_fields_attributes(self):
-        html_fields = [('ir.ui.view', 'ir_ui_view', 'arch_db', True)]
+    def _get_html_fields(self):
```

Comment:
@odoo/rd-website 
do you have any idea that why we use SQL query for field info instead of iterating the registry? If it is slow, maybe we can also ORM cache it